modules/ding_nodelist/config.rb

```diff
@@ -0,0 +1,27 @@
+# First, require any additional compass plugins installed on your system.
```

Comment:
If ding_nodelist styles are moved into ddbasic theme, I think we don't need to keep this file anymore.

Reply:
👍